.grabl/automation.yml

```
@@ -80,7 +134,7 @@ build:
         bazel test //tests:test_cluster_failover --test_output=errors
     deploy-pip-snapshot:
       image: graknlabs-ubuntu-20.04
-      dependencies: [build, test-behaviour, test-cluster-failover]
+      dependencies: [build, test-behaviour-connection, test-behaviour-concept, test-behaviour-match, test-behaviour-writable, test-behaviour-definable test-cluster-failover]
```

This is invalid automation.yml right?
```suggestion
      dependencies: [build, test-behaviour-connection, test-behaviour-concept, test-behaviour-match, test-behaviour-writable, test-behaviour-definable, test-cluster-failover]
```